content/blog/pulumi-service-improvements_02-2020/index.md

```diff
@@ -0,0 +1,104 @@
+---
+title: "Pulumi Console Improvements, February 2020"
+h1: "Pulumi Console Improvements, February 2020"
```

Comment:
These are optional, so can be removed unless they need to be different.

```suggestion
```

Do note that we have a generator that makes nice, clean Markdown files for new blog posts. You are kindly encouraged to use it. 😄

Reply:
TIL!